parsons/ngpvan/van_connector.py

```diff
@@ -1,46 +1,32 @@
 import logging
+import typing
 
-from suds.client import Client
-
-from parsons.utilities import check_env
-from parsons.utilities.api_connector import APIConnector
+from parsons.utilities.api_connector_next import APIConnector
 
 logger = logging.getLogger(__name__)
 
-URI = "https://api.securevan.com/v4/"
-SOAP_URI = "https://api.securevan.com/Services/V3/ListService.asmx?WSDL"
 
+class VANConnector(APIConnector):
```

Comment:
the van connector class is the one that knows about pagination and data unboxing.